lib/rules/no-async-in-computed-properties.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * @fileoverview Check if there are no async inside computed properties.
+ * @author Armano
+ */
+'use strict'
+
+const utils = require('../utils')
+
+const PROMISE_FUNCTIONS = [
+  'then',
+  'catch',
+  'finally'
+]
+
+const PROMISE_METHODS = [
+  'all',
+  'race',
+  'reject',
+  'resolve'
+]
+
+const TIMED_FUNCTIONS = [
+  'setTimeout',
+  'setInterval',
+  'setImmediate',
+  'requestAnimationFrame'
+]
+
+function isTimedFunction (node) {
+  return (
+    node.callee.type === 'Identifier' &&
```

Comment:
Add additional `node.type === 'CallExpression'`. It's good to always check the given node too, and not depend on implementation that is outside this function.